Sources/SourceKitLSP/Swift/SwiftTestingScanner.swift

```diff
@@ -231,6 +245,7 @@ final class SyntacticSwiftTestingTestScanner: SyntaxVisitor {
     let memberScanner = SyntacticSwiftTestingTestScanner(
       snapshot: snapshot,
       allTestsDisabled: attributeData?.isDisabled ?? false,
+      isScanningExtension: node is ExtensionDeclSyntax,
```

Comment:
Let’s use the swift-syntax casting operators which will continue to work even if `node` was changed to not be an existential at some point. Same below.

```suggestion
      isScanningExtension: node.is(ExtensionDeclSyntax.self),
```